pkg/network/render_test.go

```diff
@@ -95,6 +95,23 @@ func TestIsChangeSafe(t *testing.T) {
 	err = IsChangeSafe(next, prev)
 	g.Expect(err).NotTo(HaveOccurred())
 
+	// But you can't go from single-stack IPv4 to dual-stack IPv6-primary
+	next = OpenShiftSDNConfig.Spec.DeepCopy()
```

Comment:
You should probably start with an ovn-kubernetes config here. I think an sdn config with IPv6 addresses won't even validate anyway

Reply:
I addressed this and I reordered the test for better readability. I also fixed a typo in one of the error messages:
~~~
-               return []error{errors.Errorf("cannot change migration network type after migration is start")}
+               return []error{errors.Errorf("cannot change migration network type after migration has started")}
~~~